### ISSUE TITLE
Add commands from the PS2 version

### ISSUE THREAD

**Issue by ginggs** [marcIhm/yabasic#39]
I've seen several archives of Yabasic examples available for download.
Unfortunately, many of them use commands that were added in the PS2 version and do not run unmodified on the current version.

Would you be consider implementing the missing commands?

I believe these are some of them:
```
// Additions introduced with the PS2 version
void triangle(struct command *cmd);
void gtriangle(struct command *cmd);
void setrgb(struct command *cmd);
void setdrawbuf(struct command *cmd);
void setdispbuf(struct command *cmd);
```

**Comment by TheMeq** [comment]
This was brought up quite a bit on the old forums at yabasic.de and it was always said that this would be an issue to implement with the current graphics window.

**Comment by marcIhm** [comment]
Well, setdrawbuf and setdispbuf sound like double-buffering, which would be a useful addition. In principle I would like to implement this, but I am unsure about the amount of work involved.
About the other commands triangle, gtriangle and setrgb I am unsure, because I do not know, what they do on the ps2. And triangle is already part of ordinary yabasic.

**Comment by TheMeq** [comment]
Yes, that is double buffering. setrgb is the same as settings colors for graphics, but it allowed you to set the background color, color 1, color 2 and color 3. Color 2 and color 3 were used for the gtriangle command, which created a gradient triangle. See image below from the ps2 yabasic emulator.

![rotating_triangle](https://user-images.githubusercontent.com/7303969/72685650-fe7fd880-3ae3-11ea-82b1-e7c05254ed24.jpg)

**Comment by TheMeq** [comment]
Double buffering alone will open up lots of options for Yabasic.

**Comment by marcIhm** [comment]
Yep, double buffering will come, but not very soon. These Gradients look cool, but I think this will need support by the underlying graphics library (X11 or Win32), which I am not aware of ...

**Comment by marcIhm** [comment]
Still worth the effort.
